RELEASE_NOTES.md

```diff
@@ -27,6 +36,16 @@ Any new bugs, requests, or missing features should be reported as [Github issues
  * Remove `zcbor_any_encode()` since it was just an alias for `zcbor_nil_put()`.
  * Rename `zcbor_string_type_t` to `struct zcbor_string`.
  * Rename `zcbor_state_backups_t` to `struct zcbor_state_constant`.
+ * Add support for decoding and encoding `undefined` CBOR values.
+ * zcbor_any_decode():
+   * Rename to zcbor_any_skip()
+   * Add support for indeterminate-length arrays.
+   * Have zcbor_any_skip() consume any tags that are present.
```

Comment:
nit: Inline syntax for function calls here?

Reply:
done